chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -0,0 +1,199 @@
+# Pre-trained models for text-to-speech
+
+Text-to-speech task (also called _speech synthesis_) comes with a range of challenges.
+
+First, just like in the previously discussed automatic speech recognition, the alignment between text and speech can be tricky.  
+The model has to learn to generate the correct duration and timing for each phoneme, word, or sentence which can be challenging, 
+especially for long and complex sentences.
+
+Next, there's the long-distance dependency problem: language has a temporal aspect, and understanding the meaning of a 
+sentence often requires considering the context of surrounding words. Ensuring that the TTS model captures and retains 
+contextual information over long sequences is crucial for generating coherent and natural-sounding speech.
+
+Finally, training TTS models typically requires pairs of text and corresponding speech recordings. On top of that, to ensure 
+the model can generate speech that sounds natural for various speakers and speaking styles, data should contain diverse and 
+representative speech samples from multiple speakers. Collecting such data is expensive, time-consuming and for some languages 
+is not feasible. You may think, why not just take a dataset designed for ASR (automatic speech recognition) and use it for 
+training a TTS model? Unfortunately, automated speech recognition (ASR) datasets are not the best option. The features that 
+make it beneficial for ASR, such as excessive background noise, are typically undesirable in TTS. It's great to be able to 
+pick out speach from a noisy street recording, but not so much if your voice assistant replies to you with cars honking 
+and construction going full-swing in the background. Still, some ASR datasets can sometimes be useful for fine-tuning, 
+as finding top-quality, multilingual, and multi-speaker TTS datasets can be quite challenging.
+
+Compared to ASR (automatic speech recognition) and audio classification tasks, there are significantly fewer pretrained 
+model checkpoints available. On the Hugging Face Hub, you'll find close to 300 suitable checkpoints. Among 
+these pretrained models we'll focus on two architectures that are readily available for you in the 🤗 Transformers library - 
+SpeechT5 and Massive Multilingual Speech (MMS).
+
+## SpeechT5 
+
+SpeechT5 is a model that is capable of handling a range of speech tasks. While in this unit, we focus on the text-to-speech aspect, 
```

Comment:
Let's link the paper here since we reference it later below:
```suggestion
[SpeechT5](https://arxiv.org/abs/2110.07205) is a model published by Junyi Ao et al. from Microsoft that is capable of handling a range of speech tasks. While in this unit, we focus on the text-to-speech aspect, 
```